xml/System.IO/Path.xml

```diff
@@ -2127,13 +2121,14 @@ A trailing period in `path` is not considered an extension.
           <format type="text/markdown"><![CDATA[  
   
 ## Remarks  
- The array returned from this method is not guaranteed to contain the complete set of characters that are invalid in file and directory names. The full set of invalid characters can vary by file system. For example, on Windows-based desktop platforms, invalid path characters might include ASCII/Unicode characters 1 through 31, as well as quote ("), less than (\<), greater than (>), pipe (&#124;), backspace (\b), null (\0) and tab (\t).  
+
+This API is obsolete. Please use <xref:System.IO.Path.GetInvalidPathChars()> or <xref:System.IO.GetInvalidFileNameChars()> instead.
```

Comment:
This info is already shown prominently. Do you think we need to add it here too?

![image](https://user-images.githubusercontent.com/24882762/110854078-47a36680-8269-11eb-89b2-3415d5333bb4.png)

Reply:
Oh, I didn't know. This was meant to fix https://github.com/dotnet/dotnet-api-docs/issues/801 with a suggestion by Maira.

I suspect issue https://github.com/dotnet/dotnet-api-docs/issues/801 was created before we showed the "Obsolete" attribute string as a "caution" message in MS Docs. Is this a correct assumption?

To answer your question, it's not necessary, so I'll remove it.